### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by uladzimirzhyhar**
task4 - last.fm api

### CODE REVIEW

File: lastFm/script/albumGetInfo.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+function AlbumGetInfo() {}
+
+var reqParamArtist = 'album.getinfo&artist=';
```
Comment (NikitaSedyx) [inline lastFm/script/albumGetInfo.js]: they are in global scope, it's better to use module pattern or static

File: lastFm/script/loader.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+function Loader() {}
+
+Loader.prototype.load = function (queryParams) {
+    var API_ROOT_URL = 'http://ws.audioscrobbler.com/2.0/?method=';
+    var API_KEY = '&api_key=141d71bf2600cfe131eac31148318134';
+    var RESPONSE_FORMAT = '&format=json';
+
+    var result;
+    var url_string = API_ROOT_URL + queryParams + API_KEY + RESPONSE_FORMAT;
```
Comment (NikitaSedyx) [inline lastFm/script/loader.js]: it will be great to have encoding for params

**Comment by NikitaSedyx** [comment]
it's approve for api task, but not for arrays and binary search
I don't know why you are create branch not from master...

**Comment by uladzimirzhyhar** [comment]
What should i do to get approvement for previous tasks?

**Pull request merged**